### DISCUSSION TITLE
new MONAIBundle app

### DISCUSSION THREAD

**Discussion by RotoDrache** [Project-MONAI/MONAILabel#1488] [Q&A]
The new MONAIBundle apps do not seem to include the previous segmentation and deepedit apps. So is it not possible to train new models from scratch usinf your own data sets? IF there is still a way can you explain.
Thanks

**Comment by diazandr3s** [comment]
Hi @RotoDrache,

Thanks for asking. This is a great question.
MONAI Bundle is a specification/format of portable described deep learning models. You can learn more about this here: https://docs.monai.io/en/stable/mb_specification.html

MONAI Label supports all MONAI Bundle features: training, inference, batch inference, etc.

`The new MONAIBundle apps do not seem to include the previous segmentation and deepedit apps.`

It does include all these models. You can find an exhaustive list here: https://github.com/Project-MONAI/model-zoo/tree/dev/models

`So is it not possible to train new models from scratch usinf your own data sets?`

Yes, it is possible. Each model has instructions in the Readme file on how to do training and inference. Here is an example: https://github.com/Project-MONAI/model-zoo/tree/dev/models/wholeBody_ct_segmentation#monai-bundle-commands

Hope this helps,
